apps/condo/domains/meter/schema/MeterImportTask.js

```diff
@@ -0,0 +1,175 @@
+/**
+ * Generated by `createschema meter.MeterImportTask 'status:Select:processing,completed,canceled,error; format:Select:excel,csv; importedRecordsCount:Integer; totalRecordsCount:Integer; file?:File; errorFile?:File; user:Relationship:User:CASCADE;meta?:Json'`
+ */
+
+const { canOnlyServerSideWithoutUserRequest } = require('@open-condo/keystone/access')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const { IMPORT_FORMAT_VALUES, IMPORT_STATUS_VALUES, PROCESSING, METER_IMPORT_TASK_FOLDER_NAME } = require('@condo/domains/common/constants/import')
+const FileAdapter = require('@condo/domains/common/utils/fileAdapter')
+const { getFileMetaAfterChange } = require('@condo/domains/common/utils/fileAdapter')
+const access = require('@condo/domains/meter/access/MeterImportTask')
+const { importMeters } = require('@condo/domains/meter/tasks/index')
+
+const MeterImportTaskFileAdapter = new FileAdapter(METER_IMPORT_TASK_FOLDER_NAME)
+const setFileMetaAfterChange = getFileMetaAfterChange(MeterImportTaskFileAdapter, 'file')
+
+const MeterImportTask = new GQLListSchema('MeterImportTask', {
```

Comment:
Is it about meter readings?
```suggestion
const MeterReadingsImportTask = new GQLListSchema('MeterReadingsImportTask', {
```